src/span_context.js

```diff
@@ -68,7 +68,12 @@ export default class SpanContext {
 
   get traceId(): any {
     if (this._traceId == null && this._traceIdStr != null) {
-      this._traceId = Utils.encodeInt64(this._traceIdStr);
+      const safeTraceIdStr = this._traceIdStr.length % 2 == 0 ? this._traceIdStr : '0' + this._traceIdStr;
+      const tmpBuffer = new Buffer(safeTraceIdStr, 'hex');
```

Comment:
This should be [`Buffer.from(safeTraceIdStr, 'hex');`](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_string_encoding)